app/src/main/java/app/cash/paykit/sample/ui/ui/main/AddToCartFragment.kt

```diff
@@ -25,6 +25,7 @@ import androidx.fragment.app.Fragment
 import androidx.fragment.app.activityViewModels
 import app.cash.paykit.sample.databinding.FragmentAddToCartBinding
 
+@Suppress("ktlint:standard:backing-property-naming")
```

Comment:
We're following the standards for ViewBinding properties ( https://developer.android.com/topic/libraries/view-binding#fragments ) but it seems it clashes with this `ktlint` rule. Given ViewBinding is mostly used now since most new projects are using Compose. If we migrate ever migrate we can disable this.